packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -82,27 +82,29 @@ export default function PackagePreview({
       >
         <div className={styles.content}>
           <div className={styles.contentInner}>
-            <div className={styles.stat}>
-              <div className={styles.statHeader}>
-                <Icon kind='script' color='#8E8AA0' className={styles.statIcon} />
-                Script
+            {(pkg.containingScripts?.length ?? 0) > 0 && (
```

Comment:
nit: We're usually sticking to this pattern across the codebase `!!pkg.containingScripts?.length`

Reply:
It should also work without `!!`